packages/prices-api/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "lib": ["dom", "esnext"],
```

Comment:
do we need `dom` here?

![image](https://github.com/user-attachments/assets/c099b755-e950-4ab3-92cd-76056f0f39db)

Reply:
I live in the past where fetch used to be part of dom, but it's a global now, so it's removed 👍